src/main/webapp/Portal.html

```diff
@@ -167,6 +167,10 @@
       loadCss(cdnEndpoint + 'generated/swc.css')
       loadJs(cdnEndpoint + 'generated/jquery.min.js')
       loadJs(cdnEndpoint + 'generated/jquery-ui.min.js')
+      // Load latest version of Bootstrap 3 to override the version bundled by GWTBootstrap3
+      // The Bootstrap 3 custom theme CSS is loaded through swc.scss
+      // Note: we alias to bootstrap3 because we rely on Bootstrap 4 to be in node_modules/bootstrap to compile our scss (via synapse-react-client).
+      loadJs(cdnEndpoint + 'node_modules/bootstrap3/dist/js/bootstrap.min.js')
```

Comment:
Is node_modules available in the webapp output directory?
If it was, I would expect this asset (for example) to load properly: https://cdn-www.synapse.org/node_modules/plotly.js/dist/plotly.min.js

Reply:
You're right, I skipped the copy step